docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,100 @@
+.. _k8s-resource-caps:
+
+#####################################
+ Manage Workspace Namespace Bindings
+#####################################
+
+.. note::
+
+   Managing resource quotas in Determined requires the Determined Enterprise Edition.
+
+Determined makes it easier to manage resource quotas by handling them directly, so you don't need to
+manage them separately in Kubernetes. You can manage workspace namespace bindings and resource
+quotas using either the WebUI or CLI.
+
+*******
+ WebUI
+*******
+
+#. As an admin user, either create a new workspace or edit an existing workspace.
+
+#. In the "Namespace Bindings" section, type a name for the namespace or leave it blank. If left
+   blank, the workspace will be bound to the default namespace configured in the Master
+   Configuration YAML file.
+
+#. Toggle the "Auto Create Namespace" option on or off. When enabled, you can edit the resource
```

Comment:
Lets add that when enabled, determined will auto-create a namespace in the cluster